### PR TITLE
Tools

### PR DESCRIPTION
**Pull request by tianshuo78520a**
test=develop
New features have been added
If you want to modify the API documentation, you need to find Shanyi15 audit

### CODE REVIEW

File: paddle/scripts/paddle_build.sh [hunk 1]
```diff
@@ -532,6 +532,22 @@ function assert_api_spec_approvals() {
...
+    check_doc = `python ${PADDLE_ROOT}/tools/check_doc_approval.py`
+    if [ -n ${check_doc} ]; then
+        APPROVALS=`curl -H "Authorization: token ${GITHUB_API_TOKEN}" https://api.github.com/repos/PaddlePaddle/Paddle/pulls/${GIT_PR_ID}/reviews?per_page=10000 | \
+        python ${PADDLE_ROOT}/tools/check_pr_approval.py 1 35982308`
+        echo "current pr ${GIT_PR_ID} got approvals: ${APPROVALS}"
+        if [ "${APPROVALS}" == "FALSE" ]; then
+            echo "You must have shanyi15 approval for the api doc change! "
+            exit 1
+        fi
+        echo $check_doc >${PADDLE_ROOT}/tools/doc_md5.txt
```
Comment (reyoung) [inline paddle/scripts/paddle_build.sh]: doc_md5 is not commited?

File: tools/check_doc_approval.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+    "paddle.fluid.initializer",
+    "paddle.fluid.io",
+    "paddle.fluid.layers",
+    "paddle.fluid.metrics",
+    "paddle.fluid.nets",
+    "paddle.fluid.optimizer",
+    "paddle.fluid.profiler",
+    "paddle.fluid.recordio_writer",
+    "paddle.fluid.regularizer",
+    "paddle.fluid.transpiler"
```
Comment (luotao1) [inline tools/check_doc_approval.py]: lack of `paddle.fluid`?

Comment (tianshuo78520a) [reply tools/check_doc_approval.py]: Oh !  I think you are right, I miss the  paddle.fluid .

File: paddle/scripts/paddle_build.sh (hunk 1 above)
Comment (tianshuo78520a) [reply paddle/scripts/paddle_build.sh]: doc_md5  will Automatic generation

**Comment by shanyi15** [comment]
close via #15104

**Pull request closed without merging**